### ISSUE TITLE
Feature 1050 - Mobile menu needs padding and vertical spacing

### ISSUE THREAD

**Issue by nicktrosnant** [u3a-siteworks-development/u3a-siteworks-theme#18]
On mobile phones the menu is tight against the LH side of the screen and needs a bit more vertical spacing between items to make selection easier.
![image](https://github.com/u3a-siteworks-development/u3a-siteworks-theme/assets/34215865/ef4d1117-f832-4816-a8d7-76db794abccb)

**Comment by mike99christie** [comment]
I guess that a fix to add some padding in css would do.
This seems to be the div that needs the padding (of course just padding-left would do.
<div class="wp-block-navigation__responsive-container-content" data-wp-watch="callbacks.focusFirstElement" id="modal-1-content" style="padding: 20px;">
I don't see any issue with spacing, even with my clumsy thumbs.
Mike

**Comment by nicktrosnant** [comment]
Actual photo on my Pixel phone.  Yes, I can hit reasonably accurately but I still think a little more vertical spacing between menu items would be helpful.
![sw-menu](https://github.com/u3a-siteworks-development/u3a-siteworks-theme/assets/34215865/97dd6e93-eda0-430d-a59b-03f1bbb6c24e)

**Comment by stratplusft** [comment]
The style is in style.min.css which is in the wp-includes>blocks>navigation. Changing the padding in .wp-block-navigation ul li from 0 to 0.5rem increases the spacing without producing a menu that requires scrolling on my example site. 
Any comments before I go ahead with this change?
I've tried to attach images - let me know if you can't see them - 



![Image](https://github.com/user-attachments/assets/8c884dcf-9404-49cd-8c73-455f64ed1ecd)
![Image](https://github.com/user-attachments/assets/8059f32e-5438-4554-8be6-5865bb694fbd)

**Comment by nicktrosnant** [comment]
Looks good to me.  :-)

**Comment by mike99christie** [comment]
Here is the planned nlu3a.org.uk menu without the proposed padding change.
I don't see any good reason for extending the top/bottom padding. Even with my fat fingers, I can easily select the correct submenu options. If we do change I'd propose a very slight top/bottom padding.
![menu on mobile](https://github.com/user-attachments/assets/24e93995-74d4-423f-92e8-b6e2dc1ccbfa)
Say 5px each. Is it possible to reduce the gap between a top level menu item and the first sub level memu item.

**Comment by mike99christie** [comment]
I'll be away on holiday for two weeks now so won't be able to comment further.

**Comment by stratplusft** [comment]
The image from mike99christie fails to show that there is no gap on the left hand side.
The requirement is specific so some increase in spacing is required. The amount needed is debatable. My opinion is 0.5rem top and bottom meets the requirement, but that is subjective .
As for the start of sub menu spacing, perhaps the additional spacing doesn't need to be applied to the first item?

**Comment by nicktrosnant** [comment]
I agree with Bill's proposal regarding removing the extra spacing for the first item in a sub-menu.

Mike ... I wonder if you are actually checking on a "real" mobile phone?  Perhaps the mobile simulation on a big screen is giving you a misleading impression of the spacing?  However I'd be happy with a compromise on a small reduction on the 0.5rem that Bill proposes.

**Comment by mike99christie** [comment]
The image is from a laptop emulating a phone, but my testing is on my mobile phone.
Happy to go with your and Bill's revised proposals, with a small reduction, say to 0.3 rem

Nevertheless this is ideally really something that WordPress core should address as an ease-of-use issue, rather than being done by us!

**Comment by stratplusft** [comment]
The spacing for the sub menu seems to be coming from the agent style sheet. I don't think it makes a lot of difference whether or not the .3rem padding is included. I'd prefer to stick to the original plan for now otherwise I'll be attempting to target the submenu container with possibly unintended affects.

**Comment by stratplusft** [comment]
This is the screen image (simulatedin Chrome) using the .3rem top and bottom and .5rem left spacing

![Image](https://github.com/user-attachments/assets/920a41bd-6426-4f75-94b3-f40e76dec81d)
